pkg/sql/sqlbase/cancel_checker.go

```diff
@@ -43,19 +37,21 @@ func NewCancelChecker(ctx context.Context) *CancelChecker {
 
 // Check returns an error if the associated query has been canceled.
 func (c *CancelChecker) Check() error {
-	if !c.isCanceled && c.callsSinceLastCheck%cancelCheckInterval == 0 {
+	// Interval of Check() calls to wait between checks for context
+	// cancellation. The value is a power of 2 to allow the compiler to use
+	// bitwise AND instead of division.
+	const cancelCheckInterval = 1024
+
+	if c.callsSinceLastCheck%cancelCheckInterval == 0 {
 		select {
 		case <-c.ctx.Done():
-			c.isCanceled = true
+			// Once the context is cancelled, we no longer increment
```

Comment:
The linter here wants "canceled".

Reply:
Roger. Done. Funny that I think I asked for that linter.